Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -14,29 +14,20 @@
 namespace Leap {
   public class HandPool :
     HandFactory
-
   {
-    public IHandModel LeftGraphicsModel;
-    public IHandModel RightGraphicsModel;
-    public IHandModel LeftPhysicsModel;
-    public IHandModel RightPhysicsModel;
+
+    [SerializeField]
+    public List<IHandModel> ModelCollection;
     public List<IHandModel> ModelPool;
```

Comment:
It might be a good idea to consider making this private so it doesn't show up in the inspector.  We run the risk of having users modify the pool manually, which might cause undesired behavior